### ISSUE TITLE
Support using package IO as an alternative to curl

### ISSUE THREAD

**Issue by fingolfin** [gap-packages/qaos#3]
On a system where curl is not available, but the io package is, one could download the relevant files via io.

**Comment by olexandr-konovalov** [comment]
Agree - even more, one could support only the IO way of doing this, since the IO package is widely supported including Windows.

**Comment by fingolfin** [comment]
However, more and more sites switch to use https, which is not supported by `io` and never will be. 

Perhaps https://github.com/gap-packages/curlInterface could be an interesting alternative, but that would require somebody (e.g. @ChrisJefferson) to "finish" and deposit it, so that it is shipped with GAP. And even then, of course, it's only usable if compiled by the user (which in turn requires the presence of libcurl build headers; etc.)

**Comment by olexandr-konovalov** [comment]
@fingolfin yes, this issue evolved much since its posting - not sure we need to keep it open. 

For the record, GAPDoc got `GetByWgetOrCurl` (https://github.com/frankluebeck/GAPDoc/issues/26) - could be that https://github.com/gap-packages/curlInterface will be also of some further use there.
